### ISSUE TITLE
Evaluate usefulness of this package again

### ISSUE THREAD

**Issue by mnmkng** [apify/http-request#37]
Do we really need to maintain a separate package? We should test case sensitive headers with the latest `axios` and if it works, we should just deprecate this package. @petrpatek ?

**Comment by jancurn** [comment]
We also need Brotli encoding

**Comment by mnmkng** [comment]
Good point. We can use 'decompress: false' to turn off axios decompression and do decompression in another interceptor.

**Comment by petrpatek** [comment]
@mnmkng I think the motivation was also supporting the proxy URL directly without having to configure the agents and having only one HTTP library in projects. But I am definitely up to move this to `requestAsBrowser` directly if these reasons are not valid anymore.

**Comment by corford** [comment]
`axios` doesn't yet have native HTTP/2 support: https://github.com/axios/axios/issues/1175
